guides/source/i18n.md

```diff
@@ -275,7 +275,7 @@ get "/:locale" => "dashboard#index"
 
 Do take special care about the **order of your routes**, so this route declaration does not "eat" other ones. (You may want to add it directly before the `root :to` declaration.)
 
-NOTE: Have a look at various gems which simplify working with routes: [routing_filter](https://github.com/svenfuchs/routing-filter/tree/master), [route_translator](https://github.com/enriclluelles/route_translator).
+NOTE: Have a look at the gem that simplifies working with routes:  [route_translator](https://github.com/enriclluelles/route_translator).
```

Comment:
```suggestion
NOTE: Have a look at a gem which simplifies working with routes:  [route_translator](https://github.com/enriclluelles/route_translator).
```

Reply:
@skipkayhil made the suggestions. Thank you.